lib/plug/debugger.ex

```diff
@@ -64,7 +64,7 @@ defmodule Plug.Debugger do
 
   ## Custom Banners
 
-  You may pass an MFA (`{module, function, args}`) to be invoked when an
+  You may pass a MFA (`{module, function, args}`) to be invoked when an
```

Comment:
I believe the correct here is "an MFA". We should use "an" before "MFA" because the pronunciation of the letter "M" starts with a vowel sound (em).

Reply:
I didn't know that. I'll update it. Thanks!